spec/game_spec.rb

```diff
@@ -52,4 +47,54 @@
       expect(game.current_player.mark).to eq('x')
     end
   end
+
+  context 'Checks if user can continue playing' do
+    display = Display.new
+    player1 = Player.new('x')
+    player2 = Player.new('o')
+
+    it 'when less than 9 moves have been played, can_continue_playing returns true' do
+      allow($stdin).to receive(:gets).and_return('9')
+      board = Board.new([1, 'x', 'x', 'o', 5, 6, 'o', 8, 9])
+      game = Game.new(board, display, player1, player2)
+      game.play_move
+      expect(game.can_continue_playing?).to eq(true)
+    end
+
+    it 'when 9 moves have been played, can_continue_playing returns false' do
+      allow($stdin).to receive(:gets).and_return('9')
+      board = Board.new(['o', 'x', 'x', 'o', 'x', 'o', 'x', 'o', 9])
+      game = Game.new(board, display, player1, player2)
+      game.play_move
+      expect(game.can_continue_playing?).to eq(false)
+    end
+  end
+
+  context 'Validates a user\s move' do
```

Comment:
Think you mean 'Validates a user\'s move' but I'd prefer "Validates a user's move"